### DISCUSSION TITLE
Bonsai on linux OpenCV issues

### DISCUSSION THREAD

**Discussion by dprotter** [bonsai-rx/bonsai#2011] [Q&A]
Hi all,
I followed the [Bonsai on linux installation steps](https://github.com/orgs/bonsai-rx/discussions/1101) and am able to get bonsai up and running. However, just trying to get a webcam to connect hasnt worked. My main deviation from the install was that I couldnt get opencv 2.4 to build correctly... lots and lots of errors both at the cmake and make steps, and eventually I gave up. However, I was able to build and install opencv 4.4. 
![image](https://github.com/user-attachments/assets/e7477894-31c2-439f-ae23-c2fef60d2bb7)

Also i verified the cv install as python in my base, non-anaconda environment is able to access the webcam and stream it using opencv 4.4.

Is the only solution to re-try installing opencv 2.4?

I didnt want to clutter up the main bonsai-rx github issues with a niche issue...

Thanks!

**Comment by glopesdev** [comment]
@dprotter Indeed the vision packages require OpenCV 2.4. What is the Linux distribution you are using? There are prebuilt ubuntu binaries you may be able to use.

**Reply by dprotter** [reply]
I'm on 20.04. Prebuilt would be pretty helpful, I think!
